File: Backend-POS/POS.Main/POS.Main.Business.Admin/Models/ShopSettings/WelcomeShopInfoResponseModel.cs
namespace POS.Main.Business.Admin.Models.ShopSettings;

public class WelcomeShopInfoResponseModel
{
    public string ShopNameThai { get; set; } = string.Empty;
    public string ShopNameEnglish { get; set; } = string.Empty;
    public string FoodType { get; set; } = string.Empty;
    public string? Description { get; set; }
    public string PhoneNumber { get; set; } = string.Empty;
    public int? LogoFileId { get; set; }
    public bool HasTwoPeriods { get; set; }
    public List<OperatingHourModel> OperatingHours { get; set; } = new();
}

File: Frontend-POS/RBMS-POS-Mobile-Web/src/app/core/api/models/welcome-shop-info-response-model.ts
/* tslint:disable */
/* eslint-disable */
import { OperatingHourModel } from '../models/operating-hour-model';
export interface WelcomeShopInfoResponseModel {
  description?: string | null;
  foodType?: string | null;
  hasTwoPeriods?: boolean;
  logoFileId?: number | null;
  operatingHours?: Array<OperatingHourModel> | null;
  phoneNumber?: string | null;
  shopNameEnglish?: string | null;
  shopNameThai?: string | null;
}
